scene/2d/visibility_notifier_2d.cpp

```diff
@@ -84,10 +76,12 @@ void VisibilityNotifier2D::set_rect(const Rect2 &p_rect) {
 	rect = p_rect;
 	if (is_inside_tree()) {
 		get_world_2d()->_update_notifier(this, get_global_transform().xform(rect));
```

Comment:
Should this also be not run in editor? If we have not registered the notifier with the world, there doesn't seem a need to update (indeed might this cause error).

Reply:
Thanks -- I missed this. I'll update accordingly.